spec/lib/filterer/base_spec.rb

```diff
@@ -271,7 +293,7 @@ def starting_query
             FakeQuery.new
           end
 
-          sort_option 'whoop', -> (q, matches) { q }, tiebreaker: true
+          sort_option 'whoop', -> (matches) { nil }, tiebreaker: true
```

Comment:
Unused block argument - `matches`. If it's necessary, use `_` or `_matches` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.